tools/topology/m4/muxdemux.m4

```diff
@@ -5,7 +5,23 @@ dnl Define macro for demux widget
 dnl Mux name)
 define(`N_MUXDEMUX', `MUXDEMUX'PIPELINE_ID`.'$1)
 
-dnl W_MUXDEMUX(name, mux/demux, format, periods_sink, periods_source, kcontrol_list)
+dnl CH_MAP_COEFF(coeff)
+define(`CH_MAP_COEFF',
+`	tuples."word.$1" {'
+`	SOF_TKN_CHANNEL_MAP_COEFF'		STR($2)
+`	}')
+
+dnl CH_MAP(name, id, ext_id, mask)
+define(`CH_MAP',
+`	tuples."word.$1" {'
```

Comment:
did you mean $1 here? or $2?

Reply:
so as I understand the $1 is like "name". So it should be the same "name" meaning in both CH_MAP and CHMAP_COEFF as we have actual "id" token member in the other... goes quite confusing otherwise. I'm not totally sure what is actually the meaning of this "word.xxx" syntax, can we actually parse/see that in driver/kernel side? So I copied this from your dmic pdm...